frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -117,6 +117,7 @@ function ModelActionListItem({
                   icon="play"
                   onlyIcon
                   tooltip={t`Run`}
+                  aria-label={t`Run`}
```

Comment:
This could be in another PR, but do you think it's worth changing the underlying component instead, so it will set `aria-label` to `tooltip` automatically?

But apparently, `tooltip` only applies when `as={Link}`, so I'm not totally sure how to properly make that change. But you get the idea.

Reply:
I'm not sure I'd generally expect a tooltip and an aria-label to match all the time. I'd keep them separate so there are no surprises when updating one of them

Reply:
You're right. Not all the time. This usually happens when the underlying elements having a tooltip doesn't have text inside themselves e.g. icons.